### DISCUSSION TITLE
Should we release 0.68.4?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#29] [Patches]
Released, please refer to https://github.com/reactwg/react-native-releases/discussions/37

---

# Should we release 0.68.4?

Current Release: [0.68.3](https://github.com/facebook/react-native/releases/tag/v0.68.3)

Conversations on this thread are limited:
- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. https://github.com/facebook/react-native/commit/7d42106d4ce20c644bda4d928fb0abc163580cee
2. https://github.com/facebook/react-native/commit/0f0d52067cb89fdb39a99021f0745282ce087fc2
3. https://github.com/facebook/react-native/commit/361b310bcc8dddbff42cf63495649291c894d661
4. https://github.com/facebook/react-native/commit/951538c080ef745da304fb308fa91d597e0dd98a

#### Local commits to backport to main
1.

**Comment by kelset** [comment]
so that we won't forget https://github.com/facebook/react-native/pull/34439

**Comment by tido64** [comment]
We should pick this Yoga crash fix: https://github.com/facebook/react-native/commit/361b310bcc8dddbff42cf63495649291c894d661

**Comment by retyui** [comment]
I would be great to cherry pick https://github.com/facebook/react-native/commit/951538c080ef745da304fb308fa91d597e0dd98a too

**Comment by sunnylqm** [comment]
this one plz https://github.com/facebook/react-native/commit/7d42106d4ce20c644bda4d928fb0abc163580cee
